src/plugins/filter/filter.js

```diff
@@ -82,13 +82,40 @@ var componentName = "wb-filter",
 			}
 			inptId = elm.id + "-inpt";
 
-			totalEntries = $elm.find( ( settings.section || "" ) + " " + settings.selector ).length;
+			secSelector = ( settings.section || "" ) + " ";
+			totalEntries = $elm.find( secSelector + settings.selector ).length;
+			uiTemplate = settings.uiTemplate ? document.querySelector( settings.uiTemplate ) : "";
+
+			if ( uiTemplate ) {
+				uiInpt = uiTemplate.querySelector( "input[type=search]" );
+
+				if ( uiInpt ) {
+					uiInfo = uiTemplate.querySelector( ".wb-fltr-info" );
+
+					uiInpt.classList.add( inputClass );
+					uiInpt.setAttribute( "data-" + dtNameFltrArea, elm.id );
+					uiInpt.setAttribute( "aria-controls", elm.id );
 
-			filterUI = $( "<div class=\"input-group\">" +
-				"<label for=\"" + inptId + "\" class=\"input-group-addon\"><span class=\"glyphicon glyphicon-filter\" aria-hidden=\"true\"></span> " + i18nText.filter_label + "</label>" +
-				"<input id=\"" + inptId + "\" class=\"form-control " + inputClass + "\" data-" + dtNameFltrArea + "=\"" + elm.id + "\" aria-controls=\"" + elm.id + "\" type=\"search\">" +
-				"</div>" +
-				"<p aria-live=\"polite\" id=\"" + elm.id + "-info\">" + infoFormater( totalEntries, totalEntries ) + "</p>" );
+					if ( uiInfo ) {
+						elm.infoText = uiInfo.textContent;
+						uiInfo.id = uiInfo.id || elm.id + "-info";
+						uiInfo.setAttribute( "role", "status" );
+						uiInfo.textContent = infoFormater( totalEntries, totalEntries, elm.infoText );
+					}
+
+					filterUI = uiTemplate.innerHTML;
+					uiTemplate.remove();
```

Comment:
Can we simple leave the custom filter UI where it is? That way it will allow to have the filter in a side panel, like along with some tag filter controls.